Source/Core/DolphinQt/Settings/InterfacePane.cpp

```diff
@@ -171,6 +171,14 @@ void InterfacePane::CreateInGame()
   m_checkbox_show_active_title = new QCheckBox(tr("Show Active Title in Window Title"));
   m_checkbox_pause_on_focus_lost = new QCheckBox(tr("Pause on Focus Loss"));
   m_checkbox_hide_mouse = new QCheckBox(tr("Always Hide Mouse Cursor"));
+  m_checkbox_lock_mouse = new QCheckBox(tr("Lock Mouse Cursor"));
+
+  m_checkbox_hide_mouse->setToolTip(
+      tr("Will immediately hide the Mouse Cursor when it hovers on top of the Render Widget, "
+         "otherwise "
+         "there is a delay.\nIf \"Lock Mouse Cursor\" is enabled, it will hide on Mouse locked"));
+  m_checkbox_lock_mouse->setToolTip(tr("Will lock the Mouse Cursor to the Render Widget as long as "
+                                       "it has focus. You can set hotkey to unlock it"));
```

Comment:
Should probably say "You can set a hotkey to unlock it."

Missing the word a and a period at the end.